cpu/esp32/Makefile.features

```diff
@@ -41,6 +41,10 @@ else ifneq (,$(filter esp32c3 esp32s3,$(CPU_FAM)))
   FEATURES_PROVIDED += esp_ble_esp32c3
 endif
 
+ifneq (,$(filter esp32s2 esp32s3,$(CPU_FAM)))
+  FEATURES_PROVIDED += periph_usbdev
```

Comment:
We still have to provide this at the board level - not every board has a target USB connector (e.g. my esp32-s2-wroom based [lilygo board](https://aliexpress.com/item/1005004246784287.html) does not)